library/chacha20.c

```diff
@@ -40,9 +40,14 @@
     ((uint32_t) ((value) << (amount)) | ((value) >> (32 - (amount))))
 
 #define CHACHA20_CTR_INDEX (12U)
+<<<<<<< HEAD
```

Comment:
It looks like some git merge markers have ended up in the PR by mistake (there are more of these elsewhere)